testing/dnsdist/APKBUILD

```diff
@@ -0,0 +1,58 @@
+# Contributor: Vince Mele <vmele@inoc.com>
+# Maintainer: Vince Mele <vmele@inoc.com>
+pkgname="dnsdist"
+pkgver="1.2.1"
+pkgrel=0
+pkgdesc="dnsdist is a highly DNS-, DoS-, and abuse-aware loadbalancer."
+url="https://dnsdist.org"
+arch="all"
+license="GPL2"
+depends=""
+depends_dev=""
+makedepends="$depends_dev boost-dev libedit-dev libsodium-dev luajit-dev net-snmp-dev protobuf-dev re2-dev"
+pkgusers="dnsdist"
+pkggroups="dnsdist"
+install="$pkgname.pre-install"
+subpackages="$pkgname-doc"
+source="https://downloads.powerdns.com/releases/$pkgname-$pkgver.tar.bz2"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+
+	LIBSODIUM_CFLAGS="$CFLAGS" \
+	LIBSODIUM_LIBS="-lsodium" \
+	LIBEDIT_CFLAGS="$CFLAGS" \
+	LIBEDIT_LIBS="-ledit" \
+	PROTOBUF_CFLAGS="$CFLAGS" \
+	PROTOBUF_LIBS="-lprotobuf" \
+	RE2_CFLAGS="$CFLAGS" \
+	RE2_LIBS="-lre2" \
+	./configure \
+		--build="$CBUILD" \
+		--host="$CHOST" \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--enable-unit-tests \
+		--enable-dnscrypt \
+		--enable-libsodium \
+		--enable-re2 \
+		--with-protobuf \
+		--with-boost=/usr/include \
+		--with-net-snmp \
+		--with-luajit
```

Comment:
luajit is not supported on s390x so we need to either have `arch="... !s390x"` or use `--without-luajit` on s390x.

Reply:
Is regular `lua` supported on `s390x`?

Reply:
Dropped back to `lua5.1-dev` for `s390x` and kept `luajit-dev` for everything else.